src/transformers/modeling_blenderbot.py

```diff
@@ -0,0 +1,582 @@
+import math
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn import LayerNorm
+from transformers.modeling_utils import PreTrainedModel
+from configuration_blenderbot import BlenderbotConfig
+import numpy as np
+import warnings
+
+BLENDERBOT_PRETRAINED_MODEL_ARCHIVE_MAP = {"blenderbot": "https://cdn.huggingface.co/"}
```

Comment:
@julien-c has changed this to a `BLENDERBOT_PRETRAINED_MODEL_ARCHIVE_LIST` but this can wait to be fixed at the end (no rush).

Reply:
Yes you won't need to hardcode the url to the weights, and the model id should be `facebook/blenderbot`